test/Sema/subscripting.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-run-simple-swift | %FileCheck %s
```

Comment:
Nitpick: execution tests don't belong in test/Sema/. Can you move this to test/Interpreter/ instead?